### PR TITLE
Feature/create routine to size the lateral line

### PR DESCRIPTION
**Pull request by josafatavares13**

### CODE REVIEW

File: irrigation_design_system_backend/apps/calculate_lateral_line/calculate_friction factor.py
Comment (ravellys) [inline irrigation_design_system_backend/apps/calculate_lateral_line/calculate_friction factor.py:1]: Nice!!!

create the unitest please

File: irrigation_design_system_backend/apps/calculate_lateral_line/calculate_reynolds.py
Comment (ravellys) [inline irrigation_design_system_backend/apps/calculate_lateral_line/calculate_reynolds.py:1]: Nice!!!

create the unitest, please

File: irrigation_design_system_backend/core/domain/entity/lateral_line_entity.py
Comment (ravellys) [inline irrigation_design_system_backend/core/domain/entity/lateral_line_entity.py:1]: use this like LateralLineInput

image

File: irrigation_design_system_backend/apps/calculate_lateral_line/calculate_lateral_line.py
Comment (ravellys) [inline irrigation_design_system_backend/apps/calculate_lateral_line/calculate_lateral_line.py:1]: here,

you should to create the lateralLineSize

image

File: irrigation_design_system_backend/core/constants/lateral_line.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from _decimal import Decimal
+
+class LateralLineConstants:
+  kinematic_viscosity = Decimal(0.00000101) # m²/s
+  
+class FrictorFactorConstants:
+  transition_factor = Decimal(0.04)
+  m = Decimal(0.25)
```
Comment (ravellys) [inline irrigation_design_system_backend/core/constants/lateral_line.py:8]: put the complete name of the variables please

**Review by ravellys — CHANGES_REQUESTED** [review]
please, create the methods requested in comments and create a service to calculate the HF (step 3 and 4 here https://docs.google.com/spreadsheets/d/1QYo1amxhV7rXxUm-2INXj-kYUU8J2QSZ/edit#gid=1017660306)

File: irrigation_design_system_backend/core/constants/lateral_line.py (hunk 1 above)
Comment (josafatavares13) [reply irrigation_design_system_backend/core/constants/lateral_line.py:8]: ok

File: irrigation_design_system_backend/core/domain/entity/lateral_line_entity.py
Comment (josafatavares13) [reply irrigation_design_system_backend/core/domain/entity/lateral_line_entity.py:1]: ok

**Pull request closed without merging**